pds/contracts/PackNFT.cdc

```diff
@@ -110,59 +110,84 @@ pub contract PackNFT: NonFungibleToken, IPackNFT {
         }
     }
 
-    pub resource NFT: NonFungibleToken.INFT, IPackNFT.IPackNFTToken, IPackNFT.IPackNFTOwnerOperator {
-        pub let id: UInt64
-        pub let hash: [UInt8]
-        pub let issuer: Address
+    access(all) resource NFT:
+        NonFungibleToken.INFT,
+        IPackNFT.IPackNFTToken,
+        IPackNFT.IPackNFTOwnerOperator,
+        NonFungibleToken.NFT,
+        IPackNFT.NFT
+    {
+        access(all) let id: UInt64
+        access(all) let hash: [UInt8]
+        access(all) let issuer: Address
 
-        pub fun reveal(openRequest: Bool){
+        access(NonFungibleToken.Update | NonFungibleToken.Owner) fun reveal(openRequest: Bool){
```

Comment:
revealing and opening a pack NFT requires `NonFungibleToken.Update`- or `NonFungibleToken.Owner`-authorized reference